index.html

```diff
@@ -8,51 +8,58 @@
     <title>Keenan Botes personal portfolio </title>
 </head>
 <body>
-    <div class="container">
+    <main class="container">
```

Comment:
Kindly fix the extra space around the page as shown below 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/187659716-c329286a-fae0-401f-84c8-eeabfad7f116.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187659803-98058f16-16e4-44b6-91d7-4797e643e497.png)